blog/_src/posts/2017-06-24-turing-award-50.md

```diff
@@ -0,0 +1,732 @@
+    Title: Quotes and Stories from "Turing 50"
+    Date: 2017-06-24T20:00:52
+    Tags: dear diary, by Ben Greenman
+
+The ACM recently hosted [a celebration of 50 years of the A.M. Turing award](https://www.acm.org/turing-award-50).
```

Comment:
San Francisco, at the Westin St. Francis.

(I don't want to write that down, it's in the link.)